manifests/logging.pp

```diff
@@ -40,14 +40,13 @@
 # Copyright 2013 Patrick Mooney.
 # Copyright (c) IN2P3 Computing Centre, IN2P3, CNRS
 #
-class mit_krb5::logging(
-  $default      = '',
-  $admin_server = '',
-  $kdc          = '',
-  $defaults     = '',
+class mit_krb5::logging (
+  Optional[String] $default      = undef,
```

Comment:
`default` , `admin_server` and `kdc` should be `Array[String]` instead of `String` type

Same for `defaults` that was used for very old Puppet versions...